CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## unreleased
 
 * set Clojure 1.9.0 as a provided dependency
+* support `mysql8+` adapter (sets datasource class name to `com.mysql.cj.jdbc.MysqlDataSource`)
```

Comment:
Does the `mysql8+` value come from an upstream or is used by another similar project?

If not, I'd suggest `mysql8`, since we don't know what future MySQL versions will bring.

Reply:
Yeah, I was thinking the same. Went for `+` as we don't really know if, for example, for MySQL 9.X it won't stay the same...